src/js/pages/Home.js

```diff
@@ -1,31 +1,40 @@
 import React, { Component } from 'react';
+import { getRouteProps } from 'react-static';
+import { get } from 'lodash';
+
+import SecondaryContentBanner from 'js/page_sections/SecondaryContentBanner';
+import Service311 from 'js/page_sections/Service311';
 import GlobalSearch from 'js/modules/GlobalSearch';
 import Hero from 'js/modules/Hero';
+import ExternalLink from 'js/modules/ExternalLink';
 import homepageImg from 'images/lady_bird_lake.jpg';
 
-class Home extends Component {
-
-  render() {
-    const homepageImage = {
-      file: homepageImg,
-      title: 'Lady Bird Lake walking trail',
-    }
+import jsonFileData from '__tmpdata/services';
+const services311 = get(jsonFileData, "services311", null);
 
-		return (
-			<div>
-        <Hero home={true} image={homepageImage}>
-          <span className="coa-Hero__home-preheader">Welcome to</span>
-          <h2 className="coa-Hero__home-header">Austin, TX</h2>
-          <GlobalSearch placeholder="Search for services" />
-        </Hero>
-	      <section className="wrapper coa-section">
-	        <h1> Home Page </h1>️
-	        <p>just a placeholder</p>
-	      </section>
-			</div>
-    );
+const Home = ({ allServicePages }) => {
+  const homepageImage = {
+    file: homepageImg,
+    title: 'Lady Bird Lake walking trail',
   }
 
+  const bannerContent = () => (
+    <p>
+      Alpha.austin.gov is a new website and a work in progress. For the full City of Austin website, visit <ExternalLink to="https://austintexas.gov">austintexas.gov</ExternalLink>. Learn more about the new website at <ExternalLink to="https://bit.ly/atx-digital-services">bit.ly/atx-digital-services</ExternalLink>.
+    </p>
+  )
+
+  return (
+    <div>
+      <Hero home={true} image={homepageImage}>
+        <span className="coa-Hero__home-preheader">Welcome to</span>
+        <h2 className="coa-Hero__home-header">Austin, TX</h2>
+        <GlobalSearch placeholder="Search for services" />
+      </Hero>
+      <SecondaryContentBanner content={bannerContent()} />
+      <Service311 services311={services311} />
+    </div>
+  );
 }
 
-export default Home;
+export default getRouteProps(Home);
```

Comment:
Hey I think I read that getRouteProps was being deprecated for withRouteData https://github.com/nozzle/react-static/blob/0c0e8139acf045e309bba3a9ab9b91c295f437be/CHANGELOG.md
That said, I'm not 100% on why we use this -- mostly because I don't know 100% clearly how all this works. I thought this was used to pass the data we fetch from Joplin into our components -- but I don't see joplin data here. is this to future proof it?